examples/combined_ODNP.py

```diff
@@ -0,0 +1,333 @@
+from numpy import *
+from pyspecdata import *
+import os
+import sys
+import SpinCore_pp
+import time
+from Instruments import power_control
+from datetime import datetime
+from SpinCore_pp.verifyParams import verifyParams
+from SpinCore_pp.power_helper import gen_powerlist
+from pyspecdata.file_saving.hdf_save_dict_to_group import hdf_save_dict_to_group
+import h5py
+# {{{ from run_Hahn_echo_mw.py
+fl = figlist_var()
+save_file=True
+# {{{ experimental parameters
+# {{{ these need to change for each sample
+output_name = '150uM_TEMPOL'
+IR_postproc = 'spincore_IR_v1'
+adcOffset = 26
+carrierFreq_MHz = 14.897196
+tx_phases = r_[0.0,90.0,180.0,270.0]
+amplitude = 1.0
+nScans = 1
+nEchoes = 1
+date = datetime.now().strftime('%y%m%d')
```

Comment:
this code does not change every time -- move it out of the block of things that changes every time.